<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resetPass</class>
 <widget class="QDialog" name="resetPass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="passWordLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>New password</string>
   </property>
  </widget>
  <widget class="QLabel" name="reenterLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="reEnterEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Okaybut">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preBut">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
